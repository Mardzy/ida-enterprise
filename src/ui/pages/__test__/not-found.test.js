import NotFound from "../not-found";

describe("The 404 page component", () => {

  it("renders without error", () => {
    const wrapper = mount(<NotFound/>);

    expect(toJson(wrapper)).toMatchSnapshot();
  });

});