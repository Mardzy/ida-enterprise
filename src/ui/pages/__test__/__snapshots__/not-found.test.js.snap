// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The 404 page component renders without error 1`] = `
<NotFound>
  <h1>
    404
  </h1>
  <p>
    The page you have requested does not seem to exist.
  </p>
</NotFound>
`;
